src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -59,10 +80,95 @@ def _convert_annulus(self) -> AnnulusDescription:
+                        hanger_depth=section["top_md"],
+                        settings_depth=section["base_md"],
+                        hole_diameter=data["hole_diameter"],
+                        outer_diameter=section["outer_diameter"],
+                        inner_diameter=section["inner_diameter"],
+                        inner_roughness=CASING_DEFAULT_ROUGHNESS,
+                        material=section["material"],
+                        top_of_filler=data["top_of_cement"],
+                        filler_material=CEMENT_NAME,
+                        # TODO PWPA-1970: review this fluid default with fluid actually used by SCORE file
+                        material_above_filler=FLUID_DEFAULT_NAME,
+                    )
+                )
+                i += 1
+        return casing_sections
+
+    def _convert_tubing_list(self) -> List[TubingDescription]:
+        """Create the description for the tubing list."""
+        tubing_sections = []
+        for i, data in enumerate(self.score_input.read_tubing(), start=1):
+            tubing_sections.append(
+                TubingDescription(
+                    name=f"TUBING_{i}",
+                    length=data["base_md"] - data["top_md"],
+                    outer_diameter=data["outer_diameter"],
+                    inner_diameter=data["inner_diameter"],
+                    inner_roughness=TUBING_DEFAULT_ROUGHNESS,
+                    material=data["material"],
+                )
+            )
+        return tubing_sections
+
+    def _convert_packer_list(self) -> List[PackerDescription]:
+        """Create the description for the packers."""
+        packers = []
+        for data in self.score_input.read_packers():
+            packers.append(
+                PackerDescription(
+                    name=data["name"],
+                    position=data["position"],
+                    # TODO PWPA-1970: review this fluid default with fluid actually used by SCORE file
+                    material_above=FLUID_DEFAULT_NAME,
+                )
+            )
+        return packers
+
+    def _convert_open_hole_list(self) -> List[OpenHoleDescription]:
+        """Create the description for the open hole."""
+        open_hole = []
+        start_position = Scalar(
+            max([data["shoe_md"].GetValue() for data in self.score_input.read_casings()]),
+            LENGTH_UNIT,
+            "length",
+        )
+        for i, data in enumerate(self.score_input.read_open_hole(), start=1):
+            open_hole.append(
+                OpenHoleDescription(
+                    name=f"OPEN_HOLE_{i}",
+                    length=data["final_md"] - start_position,
+                    diameter=data["hole_diameter"],
+                    inner_roughness=ROCK_DEFAULT_ROUGHNESS,
+                )
+            )
+            start_position = data["final_md"]
+        return open_hole
+
+    def _convert_casings(self) -> WellDescription:
```

Comment:
```suggestion
    def _convert_casings(self) -> CasingDescription:
```

Reply:
Is job passing?
Isn't it dectect locally?
If so, check why.